int32 box_id
geometry_msgs/PoseStamped pose
